lib/run.js

```diff
@@ -55,7 +54,7 @@ function eslint() {
       const { line, severity, ruleId, message } = msg
       const annotationLevel = levels[severity]
       annotations.push({
-        path,
+        path: (SOURCE_ROOT ? SOURCE_ROOT + "/" + path : path),
```

Comment:
I don't think this is necessary. In [your test](https://github.com/ImisDevelopers/1_011_a_infektionsfall_uebermittellung/runs/583618675), it looks like the first portion of the path is incorrectly repeated, like 
```
client/client/public
```
instead of 
```
client/public
```